sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/spice_indexer.py

```diff
@@ -265,6 +266,38 @@ def index_spice_file(spice_file: Path):
     )
 
 
+def index_spin_file(spin_file: Path):
+    """Insert spin file metadata into spin database table.
+
+    Parameters
+    ----------
+    spin_file: Path
+        The full name and path the spin file to index
+    """
+    with db.Session() as session:
+        spin_obj = SPICEFilePath(spin_file)
+        spin_metadata = spin_obj.spice_metadata
+        spin_df = pd.read_csv(spin_file)
+        params = {
+            "file_path": str(spin_file),
+            "start_date": spin_metadata["start_date"],
+            "end_date": spin_metadata["end_date"],
+            "version": spin_metadata["version"],
+            "first_spin_utc": datetime.strptime(
+                spin_df["spin_start_utc"].values.min(),
+                "%Y-%m-%d %H:%M:%S.%f",
+            ),
+            "last_spin_utc": datetime.strptime(
+                spin_df["spin_start_utc"].values.max(),
+                "%Y-%m-%d %H:%M:%S.%f",
+            ),
```

Comment:
I'm not entirely following what we will end up querying on, `start_date` or `first_spin_utc`. Will you use both, just one, ...? I'm not sure it really matters if we are adding more information to the table here, but I was thinking that this would be similar to our science files database table where we only reference whats in the filename to be able to search s3 easier.